app/models/manager_refresh/inventory_collection.rb

```diff
@@ -965,6 +965,10 @@ def batch_size
       1000
     end
 
+    def batch_size_pure_sql
+      10_000
```

Comment:
Is this something we might want to change with a setting?  Seems wrong to hard code it